ports/stm/Makefile

```diff
@@ -253,6 +263,12 @@ SRC_QSTR += $(SRC_C) $(SRC_SUPERVISOR) $(SRC_MOD) $(SRC_COMMON_HAL_EXPANDED) $(S
 # Sources that only hold QSTRs after pre-processing.
 SRC_QSTR_PREPROCESSOR +=
 
+# Bin section settings specific to the STM32H7
+ifeq ($(MCU_VARIANT),$(filter $(MCU_VARIANT),STM32H743xx))
```

Comment:
The F76/7 devices have 128k DTCM and 16k ITCM. Please add a TODO to add it. Totally ok to follow up with it.

![Screen Shot 2020-04-16 at 2 13 03 PM](https://user-images.githubusercontent.com/52649/79507351-6a42e600-7fec-11ea-8e47-713e19febbe5.png)

Reply:
Yeah I realized I missed this this morning but wanted to get this PR wrapped up. I'll do two follow up PRs ASAP, one for this and one for the documentation.